htdocs/core/class/unitsTools.class.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+class UnitsTools
```

Comment:
Why creating a class. Why not using the cunit.class.php that is already dedicated to Unit management around cunit table. I don't see what can be the goal of adding so many more methods when we can just add few method to existing cunit.class.php

Reply:
You are right !